src/Umbraco.Cms.StaticAssets/umbraco/UmbracoBackOffice/Default.cshtml

```diff
@@ -91,6 +90,7 @@
                 @Html.Raw(ImportMapValue("@umbraco-cms/backoffice/property-editor", backofficeAssetsPath + "/packages/core/property-editor/index.js")),
                 @Html.Raw(ImportMapValue("@umbraco-cms/backoffice/property-action", backofficeAssetsPath + "/packages/core/property-action/index.js")),
                 @Html.Raw(ImportMapValue("@umbraco-cms/backoffice/culture", backofficeAssetsPath + "/packages/core/culture/index.js")),
+                @Html.Raw(ImportMapValue("@umbraco-cms/backoffice/repository", backofficeAssetsPath + "/packages/core/repository/index.js")),
```

Comment:
@madsrasmussen I wonder if Umbraco should have a Html extension method for this instead.

It seems ServiceStack has `StaticImportMap` or `ImportMap`
https://docs.servicestack.net/javascript-add-servicestack-reference#import-maps
https://servicestack.net/posts/javascript

```
@Html.StaticImportMap(new() {
    ["vue"]                  = "/lib/mjs/vue.mjs",
    ["@servicestack/client"] = "/lib/mjs/servicestack-client.mjs",
    ["@servicestack/vue"]    = "/lib/mjs/servicestack-vue.mjs",
})
```

```
@Html.ImportMap(new()
{
    ["vue"]                  = ("/lib/mjs/vue.mjs",                 "/lib/mjs/vue.min.mjs"),
    ["@servicestack/client"] = ("/lib/mjs/servicestack-client.mjs", "/lib/mjs/servicestack-client.min.mjs"),
    ["@servicestack/vue"]    = ("/lib/mjs/servicestack-vue.mjs",    "/lib/mjs/servicestack-vue.min.mjs")
})
```

Reply:
Hi @bjarnef, good suggestion. Feel free to contribute with such an extension if you wish. I think we can put it somewhere in Umbraco.Web.Common perhaps.

Extra challenge if you are up for it: Considering that you can only have one importmap defined on the page, how would developers extend the map to append their own files?

(As Mads said, we can then use that functionality later on for some automatic parsing of manifest files to lift out global exports)